commons-oauth/src/main/java/eu/europeana/api/commons/oauth2/utils/OAuthUtils.java

```diff
@@ -202,10 +207,13 @@ public static void processResourceAccessClaims(String api, Map<String, Object> d
     //extract key
     String apiKey = OAuthUtils.extractApiKey(data);
     
+    //extract affiliation
+    String affiliation = (String) data.get(AFFILIATION);
```

Comment:
will trow a runtime exception if the affiliation is not present in the jwt token, which will break the api functionality

Reply:
In that case the affiliation will be null, no runtime exception will be thrown.